src/mqtt/ConneXMqttClient.py

```diff
@@ -0,0 +1,145 @@
+"""
+ConneX MQTT Client sample application.

```

Comment:
**nitpick:** We might want to change the line to:

```suggestion
ConneX MQTT Client sample code.
```
to avoid confusion with whatever we call the full sample application